README.md

```diff
@@ -34,6 +34,12 @@ Intel® Geti™ enables anyone to build computer vision AI models in a fraction
  <img src="https://github.com/user-attachments/assets/af76a999-74d2-48bd-ae31-094e88a0d2e9" width="600"/>
 </p>
 
+Geti™ covers the complete AI model lifecycle, from dataset preparation and model training to model deployment and inferencing at the edge. You can now start building AI by running model fine-tuning on Intel® Arc™ GPU B-series. Once your computer vision solution is ready for deployment, you can easily run your OpenVINO™-optimized model across the full Intel XPU portfolio. 

```

Comment:
> You can now start building AI by running model fine-tuning on Intel® Arc™ GPU B-series.

This sentence may give the impression that we only support Arc GPUs. What about "you can now start building AI by fine-tuning models (find here list of officially supported GPUs)" with _here_ linking to such a list?